components/Nav/NavMobile/NavMobile.css

```diff
@@ -0,0 +1,64 @@
+.NavMobile {
+  display: flex;
+  position: absolute;
+  background-color: var(--gray);
+  height: var(--navBarHeight);
+  width: 100%;
+}
+
+.logo {
+  position: relative;
+  max-height: 50%;
+  max-width: 50%;
+  top: 16.25px;
+  left: 2rem;
+  cursor: pointer;
+}
+
+.icon {
+  position: absolute;
+  width: 35px;
+  top: 16px;
+  right: 2rem;
+  cursor: pointer;
+}
+
+.dropdown {
+  position: absolute;
+  top: var(--navBarHeight);
+  right: 0;
+  background-color: var(--gray);
+  width: auto;
+}
+
+.ul {
+  columns: 3;
+  -webkit-columns: 3;
```

Comment:
You don't need to manually write prefixes. They get automatically added via our PostCSS config.